packages/wrangler/src/pages/deployment-tails.tsx

```diff
@@ -0,0 +1,259 @@
+import { setTimeout } from "node:timers/promises";
+import onExit from "signal-exit";
+import { printWranglerBanner } from "..";
+import { fetchResult } from "../cfetch";
+import { readConfig } from "../config";
+import { getConfigCache } from "../config-cache";
+import { FatalError } from "../errors";
+import { logger } from "../logger";
+import * as metrics from "../metrics";
+import {
+	jsonPrintLogs,
+	prettyPrintLogs,
+	createPagesTail,
+} from "../tail/createTail";
+import { translateCLICommandToFilterMessage } from "../tail/filters";
+import { requireAuth } from "../user";
+import { PAGES_CONFIG_CACHE_FILENAME } from "./constants";
+import { promptSelectProject } from "./prompt-select-project";
+import type { ConfigPath } from "..";
+import type { YargsOptionsToInterface } from "../yargs-types";
+import type { Deployment, PagesConfigCache } from "./types";
+import type { Argv } from "yargs";
+
+type Options = YargsOptionsToInterface<typeof Options> & {
+	// Global flag
+	config?: string;
+};
+const statusChoices = ["ok", "error", "canceled"] as const;
+type StatusChoice = typeof statusChoices[number];
+const isStatusChoiceList = (
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	data?: any[]
+): data is StatusChoice[] =>
+	data?.every((d) => statusChoices.includes(d)) ?? false;
+
+export function Options(yargs: Argv) {
+	return (
+		yargs
+			.positional("deployment-id", {
+				type: "string",
+				description:
+					"Deployment to tail. Specify by environment if " +
```

Comment:
Make this description more obvious that this field is optional

Reply:
+ maybe reword as `ID of the deployment to tail`

Reply:
We really need to make the deployment ID more visible too if we want to use the UUID. Right now you can only know it from the URL which isn't great, we should display it more visibly or also accept the hostname.

Reply:
Good callout (to both comments). I do want to point out that you can run `wrangler pages deployment list` to get the UUID of your last (i think) 30 deployments, but your point about this being not great UX still stands. The happy path here is to omit the deployment ID altogether and let the tool get your _latest_ deployment for you.

@WalshyDev what do you think about renaming the positional argument `deployment-id` to `deployment` and then allow for the following values:

* Deployment ID
  - e.g. `wrangler pages deployment tail 2d5ca4e4-5701-11ed-9b6a-0242ac120002`
* Deployment URL
  - e.g. `wrangler pages deployment tail https://1ec5d691.example-project-power-ballad.pages.dev/`
* Deployment short code
  - e.g. `wrangler pages deployment tail 1ec5d691`

Reply:
I think that would be amazing!

Reply:
wasn't exactly sure how the short code thingy worked, so I just did by url for now